pkg/scheduler/scheduler.go

```diff
@@ -365,6 +366,7 @@ func (sched *Scheduler) Run(ctx context.Context) {
 
 	<-ctx.Done()
 	sched.SchedulingQueue.Close()
+	klog.V(2).InfoS("scheduling queue is closed")
```

Comment:
Scheduling queue is an internal concept, not 100% sure whether it's fit here. And if we log this in `MakeNextPodFunc`, I think no need here.